src/components/app-top-bar.tsx

```diff
@@ -4,26 +4,39 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import React, { useEffect, useState } from 'react';
+
+import React, { FunctionComponent, useEffect, useState } from 'react';
 import { LIGHT_THEME, logout, TopBar } from '@gridsuite/commons-ui';
 import Parameters, { useParameterState } from './parameters';
 import { APP_NAME, PARAM_LANGUAGE, PARAM_THEME } from '../utils/config-params';
 import { useDispatch, useSelector } from 'react-redux';
-import { fetchAppsAndUrls, fetchVersion } from '../utils/rest-api';
+import {
+    fetchAppsAndUrls,
+    fetchVersion,
+    MetadataJson,
+} from '../utils/rest-api';
 import { getServersInfos } from '../rest/study';
-import PropTypes from 'prop-types';
 import { useNavigate } from 'react-router-dom';
 import { ReactComponent as PowsyblLogo } from '../images/powsybl_logo.svg';
 import AppPackage from '../../package.json';
+import { AppState } from '../redux/reducer';
+import { UserManager } from 'oidc-client';
 
-const AppTopBar = ({ user, userManager }) => {
+export type AppTopBarProps = {
+    user?: AppState['user'];
+    userManager: {
+        instance: UserManager | null;
```

Comment:
UserManager is a class, it shoudl be `typeof UserManager`

Reply:
It might be cleaner to just use unknown as anyway the link to a proper type is not done here

Reply:
Yes but I wasn't sure it was really working like that. But if it does work that's ok for me :)